### PR TITLE
Changing rydberg level

### PR DESCRIPTION
**Pull request by LucasGitQ**

### CODE REVIEW

**Review by HGSilveri — CHANGES_REQUESTED** [review]
Some initial comments

File: pulser/devices/interaction_coefficients/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""C6 coefficients for Rydberg levels between 50 and 100"""
+
+import json
+from pathlib import PurePath
+
+C6_dict = json.load(open(PurePath(__file__).parent / "C6_coeffs.json"))
```
Comment (HGSilveri) [inline pulser/devices/interaction_coefficients/__init__.py:19]: It sucks a bit that the keys are `str`, you could maybe make a new one where they are `int` instead. Something like this:
```suggestion
_json_dict = json.load(open(PurePath(__file__).parent / "C6_coeffs.json"))
C6_dict = {int(key): value for key, value in _json_dict.items()}
```

File: pulser/devices/_device_datacls.py [hunk 2]
```diff
@@ -54,30 +53,14 @@ class Device:
     max_radial_distance: int
     min_atom_distance: int
     _channels: tuple[tuple[str, Channel], ...]
-    rydberg_level: int = 60
+    rydberg_level: int
```
[the remark this answers is not in the document]
Comment (HGSilveri) [reply pulser/devices/_device_datacls.py:56]: For example, right now there is no `rydberg_level` for `Chadoq2`, so there isn't an `interaction_coeff` either.

**Review by HGSilveri — CHANGES_REQUESTED** [review]
It's quite important to make sure the units are matching. `interaction_coeff` is not `C6`, it's `C6/hbar`

File: pulser/devices/_device_datacls.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (HGSilveri) [reply pulser/devices/_device_datacls.py:56]: > For example, right now there is no `rydberg_level` for `Chadoq2`, so there isn't an `interaction_coeff` either.

Nevermind, I see that it was previously defined.

File: pulser/tests/test_devices.py [hunk 3]
```diff
@@ -43,10 +45,12 @@ def test_init():
 
 def test_mock():
     dev = pulser.devices.MockDevice
-    assert dev.dimensions == 3
+    assert dev.dimensions == 2
```
Comment (HGSilveri) [inline pulser/tests/test_devices.py:48]: It's actually `dev.dimensions == 3` now, this is here because your branch way outdated with respect to `develop`.

Comment (LucasGitQ) [reply pulser/tests/test_devices.py:48]: Oups sorry

File: pulser/devices/interaction_coefficients/__init__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""C_6/hbar (in  um^6 / us`), coeffs for Rydberg levels between 50 and 100."""
+
+import json
+from pathlib import PurePath
+
+_json_dict = json.load(open(PurePath(__file__).parent / "C6_coeffs.json"))
+C6_dict = {int(key): value for key, value in _json_dict.items()}
```
Comment (HGSilveri) [inline pulser/devices/interaction_coefficients/__init__.py:20]: nit: This is a style thing, we shouldn't name a variable with an upper-case letter (sorry, I just remembered this now). Let's go with `c6_dict` instead

File: pulser/devices/_device_datacls.py [hunk 5]
```diff
@@ -37,23 +38,21 @@ class Device:
         max_radial_distance: The furthest away an atom can be from the center
             of the array (in μm).
         min_atom_distance: The closest together two atoms can be (in μm).
-        interaction_coeff: :math:`C_6/\hbar`
-            (in :math:`\mu m^6 / \mu s`),
-            which sets the van der Waals interaction strength between atoms in
-            the same Rydberg state.
+        rybderg level : The value of the principal quantum number n
```
Comment (HGSilveri) [inline pulser/devices/_device_datacls.py:41]:
```suggestion
        rybderg_level : The value of the principal quantum number n
```

File: pulser/devices/interaction_coefficients/__init__.py (hunk 4 above)
Comment (HGSilveri) [reply pulser/devices/interaction_coefficients/__init__.py:20]: Reminder to do this too

**Review by HGSilveri — COMMENTED** [review]
Awesome, all tests passed! Only tiny things to take care of now

**Review by HGSilveri — APPROVED** [review]
Great, thanks a lot!

**Pull request merged**